docs/plugins/hashicorp-vault.md

```diff
@@ -10,6 +10,17 @@ This plugin installs hashicorp-vault with replicas:3 by default value.
 --8<-- "hashicorp-vault.yaml"
 ```
 
+### Default Configs
+
+| key              | default value                       | description                                |
+| ----             | ----                                | ----                                       |
+| chart.chart_name | hashicorp/vault                     | chart name                                 |
+| chart.timeout    | 5m                                  | this config will wait 10 minutes to deploy |
+| upgradeCRDs      | true                                | default update CRD config                  |
+| chart.wait       | true                                | whether to wait until install is complete  |
```

Comment:
"install" is a verb.

<img width="614" alt="image" src="https://user-images.githubusercontent.com/36830265/183552069-863e24b1-7f47-4fe9-811d-0cd466cc411f.png">

You could use "whether to wait until the installation is complete"